packages/compass-e2e-tests/smoke-test.ts

```diff
@@ -339,5 +226,5 @@ run()
   })
   .catch(function (err) {
     console.error(err.stack);
-    process.exit(1);
+    process.exitCode = 1;
```

Comment:
From https://nodejs.org/api/process.html#processexitcode:

> Rather than calling process.exit() directly, the code should set the process.exitCode and allow the process to exit naturally by avoiding scheduling any additional work for the event loop:

Reply:
It is like I know this and keep independently rediscovering it yet still forget.